webpack.config.js

```diff
@@ -188,13 +140,16 @@ module.exports = ( env, argv ) => {
 					} ),
 				],
 			},
-			externals,
+			// externals,
 			resolve,
 		},
 
 		// Build the test files.
 		{
-			entry: { 'googlesitekit-tests': './assets/js/googlesitekit-tests.js' },
+			entry: {
+				'googlesitekit-tests': './assets/js/googlesitekit-tests.js',
+				'e2e-utilities': './tests/e2e/e2e-utilities.js',
+			},
```

Comment:
These are not being excluded from the release zip

<details>
<summary>Dist JS files</summary>

```
        0  01-27-2020 22:17   google-site-kit/dist/assets/js/
     1844  01-27-2020 22:17   google-site-kit/dist/assets/js/ads.js
   973157  01-27-2020 22:17   google-site-kit/dist/assets/js/allmodules.js
    44280  01-27-2020 22:17   google-site-kit/dist/assets/js/chunk-googlesitekit-adminbar-05e75b29d0d33cbe4709.js
    32177  01-27-2020 22:17   google-site-kit/dist/assets/js/chunk-googlesitekit-setup-wizard-8380113159d8cc93ff03.js
    23232  01-27-2020 22:17   google-site-kit/dist/assets/js/chunk-googlesitekit-setup-wizard-proxy-9e7a8c523a2f817a665f.js
    17925  01-27-2020 22:17   google-site-kit/dist/assets/js/chunk-googlesitekit-setup-wizard~chunk-googlesitekit-setup-wrapper-734b48a8cb72b8bd8603.js
     4473  01-27-2020 22:17   google-site-kit/dist/assets/js/chunk-googlesitekit-setup-wrapper-41213a0f28b700d924d7.js
    38127  01-27-2020 22:17   google-site-kit/dist/assets/js/e2e-utilities.js
        0  01-27-2020 22:17   google-site-kit/dist/assets/js/externals/
   606422  01-27-2020 22:17   google-site-kit/dist/assets/js/googlesitekit-activation.js
    13509  01-27-2020 22:17   google-site-kit/dist/assets/js/googlesitekit-admin.js
    15874  01-27-2020 22:17   google-site-kit/dist/assets/js/googlesitekit-adminbar-loader.js
   590003  01-27-2020 22:17   google-site-kit/dist/assets/js/googlesitekit-api.js
   627705  01-27-2020 22:17   google-site-kit/dist/assets/js/googlesitekit-dashboard-details.js
   630148  01-27-2020 22:17   google-site-kit/dist/assets/js/googlesitekit-dashboard-splash.js
   644158  01-27-2020 22:17   google-site-kit/dist/assets/js/googlesitekit-dashboard.js
   601452  01-27-2020 22:17   google-site-kit/dist/assets/js/googlesitekit-module.js
   727637  01-27-2020 22:17   google-site-kit/dist/assets/js/googlesitekit-settings.js
   554202  01-27-2020 22:17   google-site-kit/dist/assets/js/googlesitekit-tests.js
   589909  01-27-2020 22:17   google-site-kit/dist/assets/js/googlesitekit-wp-dashboard.js
   528951  01-27-2020 22:17   google-site-kit/dist/assets/js/vendors~chunk-googlesitekit-adminbar-9e789bf1fd190030d0dd.js
     7947  01-27-2020 22:17   google-site-kit/dist/assets/js/vendors~chunk-googlesitekit-setup-wizard-proxy-6b85c1dd02253863adf4.js
   539595  01-27-2020 22:17   google-site-kit/dist/assets/vendor/lodash.js
    73121  01-27-2020 22:17   google-site-kit/dist/assets/vendor/lodash.min.js
   146860  01-27-2020 22:17   google-site-kit/dist/assets/vendor/moment.js
    51679  01-27-2020 22:17   google-site-kit/dist/assets/vendor/moment.min.js
   726443  01-27-2020 22:17   google-site-kit/dist/assets/vendor/react-dom.development.js
   100002  01-27-2020 22:17   google-site-kit/dist/assets/vendor/react-dom.production.min.js
    98230  01-27-2020 22:17   google-site-kit/dist/assets/vendor/react.development.js
    11814  01-27-2020 22:17   google-site-kit/dist/assets/vendor/react.production.min.js
     3797  01-27-2020 22:17   google-site-kit/dist/assets/js/externals/svgxuse.js
     1342  01-27-2020 22:17   google-site-kit/dist/assets/js/externals/wp-polyfill-element-closest.js
     8878  01-27-2020 22:17   google-site-kit/dist/assets/js/externals/wp-polyfill-fetch.js
     4392  01-27-2020 22:17   google-site-kit/dist/assets/js/externals/wp-polyfill-formdata.js
     1284  01-27-2020 22:17   google-site-kit/dist/assets/js/externals/wp-polyfill-node-contains.js
     6969  01-27-2020 22:17   google-site-kit/dist/assets/js/externals/wp-polyfill-url.js
    97849  01-27-2020 22:17   google-site-kit/dist/assets/js/externals/wp-polyfill.js

```
</details>

Reply:
Also, we should update the build to remove the `copy-vendor` gulp task and other vendor/external  built files.

<details>
<summary>Diff</summary>

```diff
--- Untitled    Tue Jan 28 09:59:31 2020
+++ Untitled2   Tue Jan 28 09:59:31 2020
@@ -1,4 +1,4 @@
-BEFORE
+AFTER
      1844  01-27-2020 22:17   google-site-kit/dist/assets/js/ads.js
    973157  01-27-2020 22:17   google-site-kit/dist/assets/js/allmodules.js
     44280  01-27-2020 22:17   google-site-kit/dist/assets/js/chunk-googlesitekit-adminbar-05e75b29d0d33cbe4709.js
@@ -20,18 +20,3 @@
    589909  01-27-2020 22:17   google-site-kit/dist/assets/js/googlesitekit-wp-dashboard.js
    528951  01-27-2020 22:17   google-site-kit/dist/assets/js/vendors~chunk-googlesitekit-adminbar-9e789bf1fd190030d0dd.js
      7947  01-27-2020 22:17   google-site-kit/dist/assets/js/vendors~chunk-googlesitekit-setup-wizard-proxy-6b85c1dd02253863adf4.js
-   539595  01-27-2020 22:17   google-site-kit/dist/assets/vendor/lodash.js
-    73121  01-27-2020 22:17   google-site-kit/dist/assets/vendor/lodash.min.js
-   146860  01-27-2020 22:17   google-site-kit/dist/assets/vendor/moment.js
-    51679  01-27-2020 22:17   google-site-kit/dist/assets/vendor/moment.min.js
-   726443  01-27-2020 22:17   google-site-kit/dist/assets/vendor/react-dom.development.js
-   100002  01-27-2020 22:17   google-site-kit/dist/assets/vendor/react-dom.production.min.js
-    98230  01-27-2020 22:17   google-site-kit/dist/assets/vendor/react.development.js
-    11814  01-27-2020 22:17   google-site-kit/dist/assets/vendor/react.production.min.js
-     3797  01-27-2020 22:17   google-site-kit/dist/assets/js/externals/svgxuse.js
-     1342  01-27-2020 22:17   google-site-kit/dist/assets/js/externals/wp-polyfill-element-closest.js
-     8878  01-27-2020 22:17   google-site-kit/dist/assets/js/externals/wp-polyfill-fetch.js
-     4392  01-27-2020 22:17   google-site-kit/dist/assets/js/externals/wp-polyfill-formdata.js
-     1284  01-27-2020 22:17   google-site-kit/dist/assets/js/externals/wp-polyfill-node-contains.js
-     6969  01-27-2020 22:17   google-site-kit/dist/assets/js/externals/wp-polyfill-url.js
-    97849  01-27-2020 22:17   google-site-kit/dist/assets/js/externals/wp-polyfill.js
```

</details>

Also, maybe `ads.js` and `allmodules.js` don't need to be (or make sense to be) entrypoints anymore either? I think the only non-chunk js files we need are `googlesitekit-*.js` + e2e utilities.